integration/simulation/p2p/in_mem_obscuro_client.go

```diff
@@ -1,11 +1,13 @@
-package host
+package p2p
```

Comment:
This struct was only used in tests, so I moved it out of `go/host`, which should be for production code only.